rls/src/main/java/io/grpc/rls/RlsProtoData.java

```diff
@@ -494,9 +419,6 @@ public Name(String service) {
 
       /** The primary constructor. */
       Name(String service, String method) {
-        checkState(
-            !checkNotNull(service, "service").isEmpty(),
```

Comment:
Do we need to still check for null? Or declare service `@Nullable`?

Reply:
@dapengzhang0 I think you missed this comment. GitHub should do something about the visibility of comments on the code left side of the diff.

Reply:
Added `@Nullable` for method. Also removed the single-arg constructor which is only used in test.